### DISCUSSION TITLE
I Want To Discuss My Favorite Note-Taking App (Not The Original Title)

### DISCUSSION THREAD

**Discussion by Mara-Li** [manunamz/jekyll-wikilinks#25] [General]
Hello.

On my searching for wiki links, I saw you cite an Obsidian discussion.
As you use wikilinks, are you an user of Obsidian?

I work on a project named YAFPA, in Jekyll. It's a free publish (website hosted) alternative, as I don't have the money for publish (and it doesn't fit my need).

My projet rely, actually, on liquid to process wikilinks. I also use a python script to convert my file.

Wikilinks is one of the feature I deal (the original developper of the liquid template is inactive), and I really will love to see your work (wikilinks and graphic bullet) integrated in my work.

The main problem I had when I first try is the "preview on hover" that disappear.

I saw an issue about it and you talk about tutorial. This tutorial will be universal?

It was the only problem I have with your plug-in (and also I need to rewrite my python code but deleting is not pretty difficult).

I any case, your work is pretty Amazing and need to be more popular!

**Comment by manunamz** [comment]
Hi there!

Thanks for your interest in this plugin and for participating in discussions! Projects are always worth working on when they are helping others and providing real value that makes the lives of those involved that more efficient and meaningful.

These repos aim to be help anyone regardless of what application they are using (syntax and workflows permitting). If you would like to discuss other applications in more detail I would recommend you head to their respective spaces. The discussion and issue tracking for this repo is for this project. 

Discussions here aim to focus on the specifics of this project and the first principles behind possible workflows and syntaxes. I do not want these repos to be mistaken as advertising for any particular app. I hope to converge on syntaxes that will help the largest number of people over the widest array of applications, especially since there isn't one canonical wikilink syntax standard (like CommonMark or GitHub Flavoured Markdown).

This might not be the optimal thing to do and is subject to a re-think. But it seems like a reasonable way to preemptively combat looming spam-bot stampedes.

Thanks again for participating! Open source projects mean so much more when there are real people at the other end of releases. And happy wikilinking!

🎋 manunamz 💚

(I have changed the title of this discussion for future visitors to more easily find this comment via discussion titles)
